src/SDKs/_metadata/sql_resource-manager.txt

```diff
@@ -1,14 +1,11 @@
-
-Type dsc to start/install the DS Consolidated Console
-
-2018-04-06 18:59:06 UTC
+2018-04-18 15:16:26 UTC
```

Comment:
Please create a `generate.ps1` similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1) and regenerate the code.

Reply:
@dsgouda when I try and run the generate.ps1 file I've added, I get "Could not find module: C:\azure-sdk-for-net\tools\SdkBuildTools\psModules\CodeGenerationModules\generateDotNetSdkCode.psm1. Please run msbuild build.proj to install the module." However, this file is in the folder and the msbuild is passing. Any ideas why?